rust/arrow-flight/src/arrow.flight.protocol.rs

```diff
@@ -499,8 +499,9 @@ pub mod flight_service_server {
     #[async_trait]
     pub trait FlightService: Send + Sync + 'static {
         #[doc = "Server streaming response type for the Handshake method."]
-        type HandshakeStream: Stream<Item = Result<super::HandshakeResponse, tonic::Status>>
```

Comment:
I wonder if you meant to make this change in this PR?

This sometimes happens to me (I think it is something about using nightly vs stable `rustfmt`)

Reply:
SELECT @@variable and SELECT DATABASE () are two different syntaxes.

Reply:
SELECT @@variable and SELECT DATABASE () are two different syntaxes.